recipes/slivar/meta.yaml

```diff
@@ -11,13 +11,15 @@ source:
   sha256: {{ sha256hash }}
 
 build:
-  number: 0
+  number: 1
   skip: True  # [osx]
 
 requirements:
   run:
     - htslib
     - zlib
+    - coreutils
+    - curl
```

Comment:
Does it need curl during runtime? It looks like this could just be in the `build` section.

Reply:
Actually, the binaries are statically linked, so I expect there doesn't need to be anything in the `run` section.

Reply:
Makes sense.  I was making minimal changes to existing recipes.  I removed the run requirements and shifted `coreutils` and `curl` to build requirements.